src/utils/getChangedStatusName.ts

```diff
@@ -1,9 +1,9 @@
-import { COMPLETED, DONE, AVAILABLE, UNASSINGED } from '@/constants/constants';
+import { COMPLETED, DONE, AVAILABLE, UNASSIGNED } from '@/constants/constants';
 export function getChangedStatusName(name: string) {
     if (name === COMPLETED) {
         return DONE;
     } else if (name === AVAILABLE) {
-        return UNASSINGED;
+        return UNASSIGNED;
```

Comment:
why do we have this change here, i dont see anything changed?

Reply:
Correcting a spelling mistake.